app/build.gradle

```diff
@@ -16,6 +16,7 @@ android {
 
         proguardFile getDefaultProguardFile('proguard-android-optimize.txt')
         proguardFile 'proguard-searchview.pro'
+        consumerProguardFiles 'proguard-appsee.pro'
```

Comment:
should be `proguardFile`.

`consumerProguardFile` indicates that the rules should be bundled with a library project